class.jetpack-twitter-cards.php

```diff
@@ -196,9 +198,12 @@ static function sharing_global_options() {
 	static function site_tag() {
 		$site_tag = get_option( 'jetpack-twitter-cards-site-tag' );
 		if ( empty( $site_tag ) ) {
-			$site_tag = ( defined( 'IS_WPCOM' ) && IS_WPCOM ) ? 'wordpressdotcom' : 'jetpack';
+			return;
+		} else if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
```

Comment:
@georgestephanis Thanks for the comments! I updated my PR based on your comments. Hopefully that's ok!